src/pages/home.tsx

```diff
@@ -0,0 +1,26 @@
+import { NextPage } from 'next'
+import React, { useEffect } from 'react'
+import Router from 'next/router'
+import {
+  LOADING_STATE,
+  useSessionToken,
+} from '../components/SessionVerification'
+
+const Home: NextPage = () => {
+  const username = useSessionToken('/jwt')
```

Comment:
Hide the path inside `useSessionToken`

Reply:
I tried to do this but couldn't..can you please guide me on how to go about it?